### DISCUSSION TITLE
Version 0.20 not working in windows 7

### DISCUSSION THREAD

**Discussion by jiegen78** [FreeTubeApp/FreeTube#5286] [Q&A]
Until 0.19 version it worked fine.
Now it gives the error "The procedure entry point DiscardVirtualMemory could not be locted in the dynamic link library KERNEL32.dll"

thanks

**Comment by absidue** [comment]
It not working is entirely expected, as mentioned in the release notes FreeTube supports Windows 10 and above now.
